config/api.yml

```diff
@@ -2336,7 +2336,17 @@
         - ems_infra_show_list
         - ems_block_storage_show_list
       - :name: create
-        :identifier: ems_infra_new
+        :identifiers:
+        - :klass: EmsInfra
```

Comment:
We should probably use the real class names here, not the aliases

Reply:
You want to put in actual provider class names into this file? I hate that idea. The point is that we can't differentiate at the moment between the cloud and infra side so unless we use these... do you have a better idea?

Reply:
No, I meant `ManageIQ::Providers::InfraManager` rather than `EmsInfra` and `ManageIQ::Providers::CloudManager` rather than `EmsCloud`.  These are the official classes in core.